tests/unit/lexicon/test_medium.py

```
@@ -0,0 +1,158 @@
+                channel=renderer,
+            )
+        )
+
+        started = [e for e in grimoire.events if isinstance(e, RiteStarted)]
+        step_rite = next(e for e in started if e.name == "choose")
+        medium_rite = next(e for e in started if e.name == "pick")
+        assert medium_rite.category == "medium"
+        assert medium_rite.parent_id == step_rite.rite_id
+
+    @staticmethod
+    def test_a_decorated_medium_introspects_as_itself():
```

Ruff (0.15.21)

[warning] Missing return type annotation for staticmethod `test_a_decorated_medium_introspects_as_itself`

Add return type annotation: `None`

(ANN205)